pyproject.toml

```diff
@@ -40,7 +40,7 @@ classifiers = [
 ]
 # Dependencies and requirements
 requires-python = ">=3.11"
-dependencies = ["ttsim-backend>=1.0a3"]
```

Comment:
Just leaving this here as a reminder not to merge before releasing ttsim 1.0a4 (or 1.0b).